config/base/params.env

```diff
@@ -4,3 +4,4 @@ caikit-standalone-image=quay.io/opendatahub/caikit-nlp:stable-994ec60
 tgis-image=quay.io/opendatahub/text-generation-inference:stable-eba83ba
 ovms-image=quay.io/opendatahub/openvino_model_server:2024.3-release-4c8c52c	
 vllm-image=quay.io/opendatahub/vllm:stable-849f0f5
+nim-state=removed
```

Comment:
not sure if here is the best place to add this.

Reply:
@spolti I believe this was the agreed upon strategy with @israel-hdez. It ties into the odh operator logic you can see here --> https://github.com/opendatahub-io/opendatahub-operator/pull/1330

Reply:
oh, ok, it is fine then :)